doc/processor.md

```diff
@@ -19,7 +19,8 @@ of the processor class. There is also an optional overridable method for queryin
 elements at a finer grain.
 
 The process method takes the requested element as input and does the analysis 
-(here detecting empty catch blocks).
+(here detecting empty catch blocks). At any time, you can interrupt the processing
+of your processor with the call of `interrupt()` method.
```

Comment:
At any time, you can interrupt the processing of the model with a call to `interrupt()` (this stops all processors, and proceeds with the next step which is usually pretty-printing the code to disk).